meta-balena-common/classes/image_types_resin.bbclass

```diff
@@ -223,6 +223,11 @@ IMAGE_CMD_resinos-img () {
     RESIN_BOOT_PN=$(parted -s ${RESIN_RAW_IMG} print | tail -n 2 | tr '\n' ' ' | awk '{print $1}')
     parted -s ${RESIN_RAW_IMG} set ${RESIN_BOOT_PN} boot on
 
+    # When creating primary rootfs partitions, we tell parted the fstype is "ext4", but this
+    # doesn't actually create a filesystem, or have any bearing on the resulting filesystem.
+    # The only effect this seems to have is specifying the MBR partition-type code, which is the
+    # same for most Linux filesystems (0x83).
+
     # resin-rootA
     if [ "${PARTITION_TABLE_TYPE}" = "msdos" ]; then
         OPTS="primary ext4"
```

Comment:
The parted man page states:
```
mkpart part-type [fs-type] start end
```
So it seems the fs-type is optional. Could you try removing both the comment and the fs-type?

Reply:
This is a good suggestion. I tried it, and it works fine.